### PR TITLE
PRIME-2170 Updates to email templates for renewals

### PR DESCRIPTION
**Pull request by johnnyl6**

### CODE REVIEW

**Review by AMiKaH — APPROVED** [review]
💯

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public override IEnumerable<EmailTemplate> SeedData
...
...E Profile page, select Absence Management from the menu on the lefthand side of the page.</li><li>In the “Enter a date range” box, use the calendar icon to enter the first day you no longer require access to PharmaNet. The date can be in the past, but do not enter a date earlier than the last time you used PharmaNet.</li><li>Leave the end date blank if you don’t intend to use PharmaNet again. If you know a date when you will return to using PharmaNet, enter as the end date.</li><li>	Click Submit.</li><li>Enter the email address of the person who sets up PharmaNet user accounts at your clinic/pharmacy/facility/HA, and share your approval. PRIME will notify them of the date(s) when you will not be using PharmaNet.</li></ul></li><li><p>Other</p><p>Action: Explain fully in your email.</p>	</li></ol><p>	Your access to PharmaNet may be deactivated if we don’t hear from you within 4 calendar days.If that happens and you require access, please contact <a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a> for instructions.</p><br><p>Thank you,</p><br>PRIME Support", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE},
```
Comment (hogmann) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:22]: Couple things, the email should read "PRIMESupport". Missing the capital S in both

the second to last statement "Your access to PharmaNet may be deactivated if we don’t hear from you within 4 calendar days." needs space between it and the last statement as per the ticket.

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ public override IEnumerable<EmailTemplate> SeedData
...
...le page, select Absence Management from the menu on the lefthand side of the page.</li><li>In the “Enter a date range” box, use the calendar icon to enter the first day you no longer require access to PharmaNet. The date can be in the past, but do not enter a date earlier than the last time you used PharmaNet.</li><li>Leave the end date blank if you don’t intend to use PharmaNet again. If you know a date when you will return to using PharmaNet, enter as the end date.</li><li>	Click Submit.</li><li>Enter the email address of the person who sets up PharmaNet user accounts at your clinic/pharmacy/facility/HA, and share your approval. PRIME will notify them of the date(s) when you will not be using PharmaNet.</li></ul></li><li><p>Other</p><p>Action: Explain fully in your email.</p>	</li></ol><p>	Your access to PharmaNet may be deactivated if we don’t hear from you within 4 calendar days.</p><p>If that happens and you require access, please contact <a href=\"mailto:PRIMESupport@gov.bc.ca\" target=\"_top\">PRIMESupport@gov.bc.ca</a> for instructions.</p><br><p>Thank you,</p><br>PRIME Support", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE},
```
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:22]: For some reason, when looking at the template in the admin screen, this last section isn't separated into two paragraphs?  
image

Comment (johnnyl6) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs:22]: right, db migration has been re-applied to fix it.

**Review by neophyte57 — APPROVED** [review]
🎉

**Pull request merged**